apps/actions/models.py

```diff
@@ -39,3 +39,10 @@ def icon(self):
             return 'clock-o'
         else:
             return 'star'
+
+    @staticmethod
+    def from_parent(action):
```

Comment:
Maybe we can use a name making the proxy relation more explicit. E.g. proxy_of()